README.md

```diff
@@ -1 +1,67 @@
 # Events calender
+
+
+Install postgres:
+apt-get install postgresql
+
+-----------------------------
+Data directory on Ubuntu
+Creating new cluster 9.5/main ...
+  config /etc/postgresql/9.5/main
+  data   /var/lib/postgresql/9.5/main
+  locale en_GB.UTF-8
+  socket /var/run/postgresql
+  port   5432
+
+User: postgres
+-----------------------------
+
+connect to the sever: 
```

Comment:
Should say connect to `connect to database server`

Reply:
I fixed it.